### DISCUSSION TITLE
Can the inference be speed up on Apple Silicon?

### DISCUSSION THREAD

**Discussion by YofarDev** [coqui-ai/TTS#3810] [General Q&A]
Hello,

I'm on macos (M2 Pro 16Go), and the generating time is like 20s for a simple sentence. Is it normal? Any way to optimize it, or do you know any other models, softwares... that would decrease the inference time? 

I'm using this model to clone a voice from a sample : 
tts = TTS("tts_models/multilingual/multi-dataset/xtts_v2").to(device)

**Comment by yukiarimo** [comment]
Any updates?
